src/validator/Tests.java

```diff
@@ -0,0 +1,55 @@
+package validator;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class Tests {
+	private static CreditCardVendorsReader reader = new CreditCardVendorsReader();
+	private static ArrayList<CreditCardVendor> vendors = reader.returnVendorArray();
+	private static CreditCardValidator validator = new CreditCardValidator();
+	private static BufferedReader br = new BufferedReader(new InputStreamReader(System.in));
+	private static CreditCardVendor vendorMasterCard = vendors.get(1);
+	
+
+	@Test
+	public void testLengthOfNumberWhenCorrect() {
+		String number = "5584239583699571";
+		assertEquals(true, validator.checkValidity(vendorMasterCard, number));
```

Comment:
1. I would suggest to use BDD style inside test + assertThat for better readability (however in this simple case it's still OK), 
BDD style example
```
// GIVEN
String number = ...

// WHEN
boolean result = validator.check....

// THEN
assertThat(result).isTrue();
```